TemplateProject/fastlane/Fastfile

```diff
@@ -0,0 +1,95 @@
+fastlane_version "2.44.1"
+fastlane_require "semantic"
+
+xcodeproj = "./ios/TemplateProject.xcodeproj"
+build_gradle = "../android/app/build.gradle"
+
+desc "Bump the iOS and Android version with a commit and tag."
+lane :bump_version do |options|
+  type = options.fetch(:type)
+  android_incrementor = AndroidVersionIncrementer.new(build_gradle)
+
+  case type
+  when "major", "minor", "patch"
+    increment_version_number(bump_type: type, xcodeproj: xcodeproj)
+    android_incrementor.increment_version_name!(type)
```

Comment:
Instead on relying on the android stuff incrementing its own version number, what do you think about explicitly setting it to whatever the iOS version is at. This would help remove the dependency on `semantic` and would ensure that we have a single source of truth for our version, and that the iOS and Android versions will stay in sync.